Content.Shared/Weapons/Melee/SharedMeleeWeaponSystem.cs

```diff
@@ -49,6 +50,7 @@ public abstract class SharedMeleeWeaponSystem : EntitySystem
     [Dependency] private readonly IPrototypeManager _protoManager = default!;
     [Dependency] private readonly StaminaSystem _stamina = default!;
     [Dependency] private readonly ContestsSystem _contests = default!;
+    [Dependency] private readonly IConfigurationManager _config = default!;
```

Comment:
Why is this added if not used, just asking.

Reply:
Probably a leftover from when the Contest Constructor lived in a Partial of SharedMeleeWeaponSystem.